src/components/form.jsx

```diff
@@ -14,13 +14,14 @@ class form extends Component {
     qualification: '',
     about: '',
     c1: null,
-    c2: null,
-    c3: null,
-    c4: null,
+    c2: '',
+    c3: '',
+    c4: '',
```

Comment:
Classes `c1, c2, c3, c4` are of type `Number`

Reply:
Actually initially, the initial state for c2,c3,c4 was null, and due to that the '/add' route was giving a validation error due to schema, i.e. it was expecting a number instead of null. That's why I made the state empty to handle that error.

Reply:
See, I'll tell you about the flow, we are getting a file from the frontend and we are sending that to the backend using a formData. Now from the backend, we use "multer" to store it in a directory with a "unique filename". Now, you've noticed that I have changed the schema in which I've included a "profilePic" which is of type string. I am storing that "unique filename" of that particular user into the "profilePic" field so that everytime anyone registers as a new user all his details (along with the name of the file which acts as a profilePic) get submitted to the database. Now I can easily use that profilePic in card.js since I know the filename and it is present at a particular path, So, I just need to give an img src for that!